opba-embedded-starter/src/main/resources/application.yml

```diff
@@ -5,7 +5,7 @@ spring:
   # Datasource connection properties
   datasource:
     # Database URL
-    url: jdbc:postgresql://localhost:5432/open_banking?currentSchema=banking_protocol
+    url: jdbc:postgresql:///open_banking?socketFactory=org.newsclub.net.unix.AFUNIXSocketFactory$FactoryArg&socketFactoryArg=/var/run/postgresql/.s.PGSQL.5432
```

Comment:
I don't think unix socket method should be default. It could be mentioned in a comment or in documentation